### PR TITLE
Update template for hazelcast package

### PR DESCRIPTION
**Pull request by frant-hartm**

### CODE REVIEW

File: hazelcast@5.X.rb [hunk 1]
```diff
@@ -16,7 +17,7 @@ def install
         (bin/executable_name).write_env_script libexec/"bin/#{executable_name}", Language::Java.overridable_java_home_env
       end
       prefix.install_metafiles
-      inreplace libexec/"bin/download/hazelcast-download.properties", "hazelcastDownloadId=CLI", "hazelcastDownloadId=CLI_BREW"
+      inreplace libexec/"lib/hazelcast-download.properties", "hazelcastDownloadId=distribution", "hazelcastDownloadId=brew"
```
Comment (ldziedziul) [inline hazelcast@5.X.rb:20]: It's a bug?

File: hazelcast@5.X.rb [hunk 2]
```diff
@@ -1,9 +1,10 @@
 class HazelcastAT5X < Formula
-    desc "Tool to run Hazelcast IMDG member instances locally"
+    desc "A tool that allows users to install & run Hazelcast"
```
Comment (ldziedziul) [inline hazelcast@5.X.rb]: This tool doesn't allow to install Hazelcast as it's already there when the package is installed.

File: README.md [hunk 3]
```diff
@@ -18,10 +18,29 @@ See [Hazelcast Cloud Command Line](https://github.com/hazelcast/hazelcast-cloud-
...
 
+Install particular version of Hazelcast
+
+    $ brew install hazelcast@5.0.1
+
+Install snapshot version of Hazelcast
+
+    $ brew install hazelcast@5.1.snapshot
+
+Install Hazelcast Enterprise edition (you need a license key to run it)
```
Comment (ldziedziul) [inline README.md]: What do you think about adding [link](https://docs.hazelcast.com/hazelcast/latest/getting-started/get-started-enterprise#installing-a-license-key) to the docs here:

**Review by ldziedziul — COMMENTED** [review]
Nice! Just a few comments

File: hazelcast@5.X.rb (hunk 1 above)
Comment (frant-hartm) [reply hazelcast@5.X.rb:20]: The CLI distribution produced previously in the `hazelcast-command-line` repo put this file into bin/download for some reason, this just uses the correct location we use in our standard artifact.

File: hazelcast@5.X.rb (hunk 2 above)
Comment (frant-hartm) [reply hazelcast@5.X.rb]: Changed the description to 

```
Hazelcast is a streaming and memory-first application platform for fast, stateful, data-intensive workloads on-premises, at the edge or as a fully managed cloud service.
```
taken from https://hazelcast.com/

**Review by ldziedziul — APPROVED** [review]

**Pull request merged**